convert2rhel/checks.py

```diff
@@ -138,14 +140,18 @@ def check_custom_repos_are_valid():
     # Without clearing the metadata cache, the `yum makecache` command may return 0 (everything's ok) even when
     # the baseurl of a repository is not accessible. That would happen when the repository baseurl is changed but yum
     # still uses the previous baseurl stored in its cache.
-    call_yum_cmd(command="clean", args="metadata")
+    call_yum_cmd(command="clean", args="metadata", print_output=False)
 
-    output, ret_code = call_yum_cmd(command="makecache", args="-v --setopt=*.skip_if_unavailable=False")
+    output, ret_code = call_yum_cmd(
+        command="makecache", args="-v --setopt=*.skip_if_unavailable=False", print_output=False
+    )
     if ret_code != 0:
         logger.critical(
             "Unable to access the repositories passed through the --enablerepo option. "
-            "See the above YUM/DNF output for more details."
+            "See the below YUM/DNF output for more details:\n{0}".format(output)
```

Comment:
Possible restructure
```suggestion
            "For more details, see YUM/DNF output:\n{0}".format(output)
```